frontend/src/Pages/HomePage.js

```diff
@@ -43,7 +121,7 @@ export default function HomePage() {
                     type="text"
                     onChange={(e) => setCode(e.target.value)}
                 ></TextField>
-                <Button variant="outlined" onClick={joinLobby} id="join-lobby">
+                <Button variant="outlined" onClick={joinLobby} id="join-lobby" disabled={joinDisabled}>
```

Comment:
Is the create-lobby button also disabled?

Reply:
no it is always enabled

Reply:
Shouldnt it also need to be disabled so that players dont create a lobby with a blank nickname

Reply:
oh yea good spot